strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -104,180 +88,19 @@ public Storage getStorage(String storageId)
         return configurationManager.getConfiguration().getStorage(storageId);
     }
 
-    @Override
-    public ArtifactInputStream getInputStream(String storageId,
-                                              String repositoryId,
-                                              String path)
-            throws IOException,
-                   NoSuchAlgorithmException
```

Comment:
@carlspring pleace notice that `getInputStream/getOutputStream` mehtods was removed from `LayoutProvider` interface. You should use `Files.newInputStream/Files.newOutputStream` now instead.